src/frontends/pytorch/src/op_table.cpp

```diff
@@ -498,6 +498,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::size", op::translate_size},
         {"aten::slice", op::quantizable_op<op::translate_slice>},
         {"aten::softmax", op::translate_softmax},
+        {"aten::softplus", op::translate_1to1_match_1_inputs<opset10::SoftPlus>},
```

Comment:
can we avoid use of `opset` namespace and specify concrete version of `Softplus`?

Reply:
This file have many imports, agree it makes sense to import specific operation, but that is not a scope of this PR.